js/data/index.d.ts

```diff
@@ -31,7 +42,7 @@ export type GroupDescriptor<T> = KeySelector<T> | BaseGroupDescriptor<T> & {
  * @type object
  * @skip
  */
-export type SortDescriptor<T> = GroupDescriptor<T>;
+export type SortDescriptor<T> = OrderingDescriptor<T>;
```

Comment:
Previously it can be `KeySelector` as well. Is this intended?

```suggestion
export type SortDescriptor<T> = KeySelector<T> | OrderingDescriptor<T>;
```

Reply:
You are right, missed it. Fixed.